docs/source/markdown/podman-run.1.md

```diff
@@ -1417,6 +1420,26 @@ $ podman run --security-opt label=type:svirt_apache_t -i -t centos bash
 
 Note you would have to write policy defining a **svirt_apache_t** type.
 
+If you want to additionaly mask specific paths in the container, you can specify
```

Comment:
```suggestion
If you want to additionally mask specific paths in the container, you can specify
```

Reply:
No "you" or similar pronouns in man pages, please.  Yet something else we're not consistent with (and a side project I need to do to clean).  Perhaps:
```suggestion
To mask additional specific paths in the container,  specify
```

Reply:
Fixed